lldb/test/API/functionalities/breakpoint/same_cu_name/common.cpp

```diff
@@ -0,0 +1,12 @@
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
```

Comment:
These stringification macros from the original bug repro are no longer needed here.